packages/flutter_tools/templates/app_shared/windows.tmpl/flutter/CMakeLists.txt

```diff
@@ -84,6 +89,7 @@ add_dependencies(flutter_wrapper_app flutter_assemble)
 # flutter tool.
 set(PHONY_OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/_phony_")
 set_source_files_properties("${PHONY_OUTPUT}" PROPERTIES SYMBOLIC TRUE)
+
```

Comment:
Migrating older versions of this file should result in the exact same file as the latest template. The migration doesn't add a new line here. We should either remove the new line here, or, update the migration to insert this new line:

```suggestion
```